mappings/net/minecraft/entity/mob/MobEntity.mapping

```diff
@@ -106,11 +122,15 @@ CLASS net/minecraft/class_1308 net/minecraft/entity/mob/MobEntity
 		ARG 2 maxYawChange
 		ARG 3 maxPitchChange
 	METHOD method_5952 setCanPickUpLoot (Z)V
+		ARG 1 pickUpLoot
 	METHOD method_5953 getPreferredEquipmentSlot (Lnet/minecraft/class_1799;)Lnet/minecraft/class_1304;
 		ARG 0 stack
 	METHOD method_5954 attachLeash (Lnet/minecraft/class_1297;Z)V
 		ARG 1 entity
-	METHOD method_5955 isBetterItemFor (Lnet/minecraft/class_1799;Lnet/minecraft/class_1799;)Z
+		ARG 2 sendPacket
+	METHOD method_5955 isBetterStackToEquip (Lnet/minecraft/class_1799;Lnet/minecraft/class_1799;)Z
```

Comment:
This is quite a verbose name, and it doesn't quite reflect that the result depends on the mob. What would you think of `isMoreDesirableEquipment` ?

Reply:
`isBetterEquipment`

Reply:
I don't like `better` as it implies some objectivity. Piglins will prefer gold items even though they are objectively worse. Actually, why not call it `prefersEquipment` ?

Reply:
`prefersNewEquipment`?